app/models/journeys/additional_payments_for_teaching/slug_sequence.rb

```diff
@@ -184,12 +184,16 @@ def personal_details_form
       end
 
       def replace_ecp_only_induction_not_completed_slugs(sequence)
+        # TODO: Remove this workaround to the shim class not reloading the answers after they're updated
+        eligibility = Policies::EarlyCareerPayments::PolicyEligibilityChecker.new(answers:)
+
+        dead_end_slug = (eligibility.status == :eligible_later) ? "eligible-later" : "ineligible"
+
```

Comment:
Not a blocker but now the shim has been removed can we just do?
```suggestion
        dead_end_slug = (ecp_eligibility_checker.status == :eligible_later) ? "eligible-later" : "ineligible"
```